.sync.yml

```diff
@@ -0,0 +1,52 @@
+---
+.travis.yml:
+  bundler_args: ' '
+  branches:
+    - release
+
+Gemfile:
+  required:
+    ':development':
+      - gem: 'nokogiri'
```

Comment:
Some of these gems aren't necessary because they will be defined in the puppet-module-gems, which are the meta gems that were created to manage gem dependencies without having to release a new version of a module each time a pin is added.

Take a look at the list of defined dependencies here: https://github.com/puppetlabs/puppet-module-gems/blob/master/config/dependencies.yml

There is a matrix defined for windows, posix, and shared for 4 different versions of Ruby. Things like metadata-json-lint, puppet-lint, puppetlabs_spec_helper.. don't need to be defined explicitly in your Gemfile anymore.

Reply:
@bmjen Thanks alot for the valuable comments. Incorporated all comments.